### PR TITLE
webgpu: remove custom size parameter of dataToGPU

### PR DESCRIPTION
**Pull request by xhcao**
To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/6393)

### CODE REVIEW

**Comment by xhcao** [comment]
@qjia7 @haoyunfeix @axinging @gyagp, please take a look, thank you.

**Review by qjia7 — APPROVED** [review]
LGTM, thanks.

Add @lina128 to take another look.

File: tfjs-core/src/tensor.ts [hunk 1]
```diff
@@ -381,6 +378,9 @@ export class Tensor<R extends Rank = Rank> {
    * For WebGL backend, the data will be stored on a densely packed texture.
    * This means that the texture will use the RGBA channels to store value.
    *
+   * For WebGPU backend, the data will be stored on a buffer. There is no
+   * parameter, so can not use an user defined size to create the buffer.
+   *
    * @param options:
    *     For WebGL,
```
[the remark this answers is not in the document]
Comment (xhcao) [reply tfjs-core/src/tensor.ts:385]: Hi, @lina128 , Please take a look, especially review the issue asked by Jiajia, thank you.

**Review by lina128 — APPROVED** [review]
LGTM

> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/6393#-:-N2CmCKz0f1c7JhSchYI:bb6ywmx)* status: :shipit: complete! 2 of 1 approvals obtained (waiting on @qjia7)

___
*[`tfjs-core/src/tensor.ts` line 385 at r2](https://reviewable.io/reviews/tensorflow/tfjs/6393#-N1layqdXxIxu_TI9E1Y-r2-385:-N2ClzBrESvMiWrETIaW:br9adm8) ([raw file](https://github.com/tensorflow/tfjs/blob/620690666651760e69d6acfc18c4eb455a14a935/tfjs-core/src/tensor.ts#L385)):*
> https://js.tensorflow.org/api/latest/#tf.Tensor.dataToGPU
Good catch! I just realized the returns field is not captured at all, not just this one. We'll fix it. Thank you!

**Pull request merged**